app.py

```diff
@@ -408,6 +437,13 @@ def _get_risk_level(debt_to_supply_ratio: float) -> str:
         if not protocol:
             streamlit.write(f'Selected random protocol = {random_protocol}.')
             protocol = random_protocol
+        # Normalize the user address by adding leading zeroes if necessary
+        user = normalize_address(user)
```

Comment:
Pls use the already existing function: https://github.com/CarmineOptions/derisk-research/blob/master/src/helpers.py#L175.